Core/AppUrls.swift

```diff
@@ -22,26 +22,29 @@ import Foundation
 public struct AppUrls {
 
     private struct Url {
-
-        // You can change this to use a subdomain for testing (e.g. "test.")
-        static let subdomain = ""
+        
+        static var devMode: String {
+            return isDebugBuild ? "?test=1" : ""
+        }
```

Comment:
We'll always send append `test=1` when running in debug mode now.  This is used by the backend to exclude data.